modules/ROOT/pages/security/authorization.adoc

```diff
@@ -7,9 +7,19 @@ They use predicates to evaluate the data accessed by the Cypher generated from a
 
 All authorization rules have an implied requirement for authentication, given that the rules are normally evaluated against values in the JWT payload.
 
-In the case of explicit authentication, configured using the `@authentication` directive, it is only ever evaluated during Cypher translation time.
+In the case of explicit authentication, configured using the xref::/security/authentication.adoc#_authentication[`@authentication`] directive, it is only ever evaluated during Cypher translation time.
 Unauthenticated requests with queries requiring authentication never reach the database.
 
+== `@authorization`
+
+=== Definition
+
+// tba
```

Comment:
```suggestion
```

As with `authorization` this is too complicated to write a simple definition for, I think.


There exists a "simple" case for `validate`:

```
[source, graphql, indent=0]
----
"""
This is a simpler version of the authorization directive to be used in the validate-document step.
"""
directive @authorization(
  """filter"""
  filter: [String]
  """validate"""
  validate: [String]
) on OBJECT | FIELD_DEFINITION
----
```